File: Reporting/CS/DevExpress.DemoReports/AnchorVertical/Report.cs
using DevExpress.XtraReports.UI;

namespace XtraReportsDemos.AnchorVertical {
    public partial class Report : XtraReport {
        public Report() {
            InitializeComponent();
            Name = ReportNames.AnchorName;
            DisplayName = ReportNames.Anchor;
        }
    }
    public class ReportWeb : Report {
        private DevExpress.XtraReports.Parameters.Parameter LandscapeParameter;
        public ReportWeb() {
            InitializeComponent();
        }
        private void InitializeComponent() {
            LandscapeParameter = new DevExpress.XtraReports.Parameters.Parameter();
            ((System.ComponentModel.ISupportInitialize)(this)).BeginInit();
            // 
            // LandscapeParameter
            // 
            LandscapeParameter.Description = "Landscape";
            LandscapeParameter.Name = "LandscapeParameter";
            LandscapeParameter.Type = typeof(bool);
            LandscapeParameter.ValueInfo = "False";
            // 
            // Report
            // 
            Parameters.AddRange(new DevExpress.XtraReports.Parameters.Parameter[] {
            LandscapeParameter});
            ExpressionBindings.AddRange(new DevExpress.XtraReports.UI.ExpressionBinding[] {
            new DevExpress.XtraReports.UI.ExpressionBinding("BeforePrint", "Landscape", "[Parameters.LandscapeParameter]")});
            ((System.ComponentModel.ISupportInitialize)(this)).EndInit();
        }
    }
}

File: Reporting/VB/DevExpress.DemoReports/AnchorVertical/Report.vb
Imports DevExpress.XtraReports.UI

Namespace XtraReportsDemos.AnchorVertical

    Public Partial Class Report
        Inherits XtraReport

        Public Sub New()
            InitializeComponent()
            Name = ReportNames.AnchorName
            DisplayName = ReportNames.Anchor
        End Sub
    End Class

    Public Class ReportWeb
        Inherits Report

        Private LandscapeParameter As DevExpress.XtraReports.Parameters.Parameter

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Overloads Sub InitializeComponent()
            LandscapeParameter = New DevExpress.XtraReports.Parameters.Parameter()
            CType(Me, System.ComponentModel.ISupportInitialize).BeginInit()
            ' 
            ' LandscapeParameter
            ' 
            LandscapeParameter.Description = "Landscape"
            LandscapeParameter.Name = "LandscapeParameter"
            LandscapeParameter.Type = GetType(Boolean)
            LandscapeParameter.ValueInfo = "False"
            ' 
            ' Report
            ' 
            Parameters.AddRange(New DevExpress.XtraReports.Parameters.Parameter() {LandscapeParameter})
            ExpressionBindings.AddRange(New ExpressionBinding() {New ExpressionBinding("BeforePrint", "Landscape", "[Parameters.LandscapeParameter]")})
            CType(Me, System.ComponentModel.ISupportInitialize).EndInit()
        End Sub
    End Class
End Namespace
